src/components/TodoItem.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes, { number } from 'prop-types';
+
+import users from '../api/users';
+
+function TodoItem({ title, completed, userId }) {
+  return (
+    <div className={completed ? 'border bg-success' : 'border bg-danger'}>
+      {users.map(user => (user.id === userId ? <h4>{user.name}</h4> : null)) }
+      <p>{title}</p>
+      {completed ? <i><b>Completed</b></i> : null}
```

Comment:
хорошим тоном є не стилізувати слова тегами, а робити це стилями
+ елементи переносити на новий рядок
```suggestion
      {completed
       ? (<i><b>Completed</b></i>) 
       : null}
```